hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -5324,22 +5328,25 @@ private long replayRecoveredEditsForPaths(long minSeqIdForTheRegion, FileSystem
   }
 
   private void handleException(FileSystem fs, Path edits, IOException e) throws IOException {
-    boolean skipErrors = conf.getBoolean(HConstants.HREGION_EDITS_REPLAY_SKIP_ERRORS,
-      conf.getBoolean("hbase.skip.errors", HConstants.DEFAULT_HREGION_EDITS_REPLAY_SKIP_ERRORS));
-    if (conf.get("hbase.skip.errors") != null) {
-      LOG.warn("The property 'hbase.skip.errors' has been deprecated. Please use "
-          + HConstants.HREGION_EDITS_REPLAY_SKIP_ERRORS + " instead.");
-    }
-    if (skipErrors) {
+    if (isSkipErrorEnabled()) {
```

Comment:
By default this skip error config is false.  So similar case of WAL split to hfiles, we will have partial Hfiles from initial split and loading of that will fail always right?  (loadRecoveredHFilesIfAny ())
cc @infraio

Reply:
Sorry sir, I don't catch you. You mean set skip error config to true?